src/components/visualizations/chart/test/loadEvent.spec.ts

```diff
@@ -0,0 +1,84 @@
+// (C) 2007-2018 GoodData Corporation
+import handleChartLoad from '../events/load';
+
+const buildChart = (
+    axisGroupAddClass: () => {},
+    labelGroupAddClass: () => {},
+    opposite: boolean
+) => ({
+    yAxis: [
+        {
+            axisGroup: {
+                addClass: axisGroupAddClass
+            },
+            labelGroup: {
+                addClass: labelGroupAddClass
+            },
+            opposite
+        }
+    ]
+});
+
+const addSecondaryAxisToChart = (
+    chart: any
+) => {
+    const { yAxis: [ primaryAxis ] } = chart;
+    return {
+        yAxis: [
+            primaryAxis,
+            {
+                ...primaryAxis,
+                opposite: !primaryAxis.opposite
+            }
+        ]
+    };
+};
+
+const executeTest = (opposite: boolean, isDualAxesChart: boolean) => {
+    const axisGroupAddClass = jest.fn();
+    const labelGroupAddClass = jest.fn();
+
+    let chart = buildChart(axisGroupAddClass, labelGroupAddClass, opposite);
+    if (isDualAxesChart) {
+        chart = addSecondaryAxisToChart(chart);
+    }
+
+    const boundHandleChartLoad = handleChartLoad.bind(chart);
+    boundHandleChartLoad();
+
+    return {
+        axisGroupAddClass,
+        labelGroupAddClass
+    };
+};
+
+describe('handleChartLoad()', () => {
+    // tslint:disable-next-line:max-line-length
+    it('should call addClass() axis methods with classname saying it\'s primary axis if \'opposite\' property is falsy', () => {
+        const { axisGroupAddClass, labelGroupAddClass } = executeTest(false, false);
+
+        expect(axisGroupAddClass).toBeCalledWith('s-highcharts-primary-yaxis');
+        expect(labelGroupAddClass).toBeCalledWith('s-highcharts-primary-yaxis-labels');
+    });
+
+    // tslint:disable-next-line:max-line-length
+    it('should call addClass() axis methods with classname saying it\'s secondary axis if \'opposite\' property is truthy', () => {
```

Comment:
This test is no longer needed. Test case below covers the case when chart is dual axis. I guess there is no way chart would have only secondary axis.

Reply:
yes, we have this case.
measures now can be on left or right axis, or both
in AD, we have new checkbox ‘show on right axis’
so all or some measures can be on right axis